src/modules/devfest-module/components/devfest-drawer/devfest-drawer.scss

```diff
@@ -9,6 +9,7 @@ app-drawer {
     background-color: #2196f3;
     padding: 24px 12px;
     box-shadow: 0px 3px 3px rgba(0, 0, 0, 0.1);
+    transform:none !important;
```

Comment:
This is not acceptable. We don't force transform because it is being changed from the app-header's behavior. This will break things if we are not thinking ahead and just using quick fix. Check app-header documentation in webcomponents.org on how to make it sticky. That's the proper way and not forcing it using `!important`

And please don't use !important on css unless it is really really needed. This feels hackish.

Reply:
Ok., I will make the necessary changes as recommended and submit a new PR today.

Reply:
Don't submit a new PR. Just push your changes here in this branch